base/nullable.jl

```diff
@@ -61,6 +61,68 @@ end
 
 get(x::Nullable) = isnull(x) ? throw(NullException()) : x.value
 
+"""
+    unsafe_get(x)
+
+Return the value of `x` for [`Nullable`](:obj:`Nullable`) `x`; return `x` for
```

Comment:
That syntax is being removed in #18588 in favour of `` [`Nullable`](@ref) ``, so probably no need to add it to the docs at this point.

> so I figured I'd include them here, too.

We can never have too many cross references :+1: